OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs

```diff
@@ -26,7 +26,7 @@ public class VideoPlayerWidget : Widget
 		public bool DrawOverlay = true;
 		public bool Skippable = true;
 
-		public bool Paused { get; private set; }
+		public bool Paused => startTime == null || pauseTime != null;
```

Comment:
this suggests that not having started is the same as paused. ok, but initially confusing.